website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -44,6 +51,14 @@ This change removes the backward-compatibility behavior introduced in Consul 1.1
 
 ## Consul 1.15.x
 
+#### Vault Enterprise as CA
+Using Vault as CA with Consul version 1.15.6 will fail to initialize the CA if [`namespace`](/consul/docs/connect/ca/vault#namespace) is set
+but [`intermediate_pki_namespace`](/consul/docs/connect/ca/vault#intermediatepkinamespace) or [`root_pki_namespace`](/consul/docs/connect/ca/vault#rootpkinamespace)
+are empty. This is a bug which will be fixed in a future version.
+
+To work around this issue, users must explicitly set [`intermediate_pki_namespace`](/consul/docs/connect/ca/vault#intermediatepkinamespace) and
+[`root_pki_namespace`](/consul/docs/connect/ca/vault#rootpkinamespace) to the same value as [`namespace`](/consul/docs/connect/ca/vault#namespace).
+
 #### Service mesh compatibility ((#service-mesh-compatibility-1-15))
 
 Upgrade to **Consul version 1.15.2 or later**.
```

Comment:
Do these all belong within a "service mesh compatibility" section (like exists for 1.15 and 1.14)?

And would it affect our upgrade guidance here? Like ... upgrade to Consul 1.15.2 or later, but avoid 1.15.6 if [using Vault Enterprise as a CA](link to anchor tag).

Reply:
I moved around the sections so they fall under "service mesh compatibility" except for 1.16 which didn't have the section.